### PR TITLE
hw-6.1-solution

### PR DESCRIPTION
**Pull request by SviatoslavExpert**

### CODE REVIEW

File: src/main/java/core/basesyntax/Pair.java [hunk 1]
```diff
@@ -1,5 +1,75 @@
...
+Pair<Integer, String> pair = Pair.of(1, "hello");
+Integer i = pair.getFirst(); // 1
+String s = pair.getSecond(); // "hello"
+
+Pair<Integer, String> pair2 = Pair.of(1, "hello");
+boolean mustBeTrue = pair.equals(pair2); // true!
+boolean mustAlsoBeTrue = pair.hashCode() == pair2.hashCode(); // true!
+ */
+class Pair<T, V> {
+    private T object1;
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Pair.java]:
```suggestion
    private T firstObject;
```
the same with other variable

File: src/main/java/core/basesyntax/Pair.java [hunk 2]
```diff
@@ -1,5 +1,75 @@
...
+
+    public void setObject1(T object1) {
+        this.object1 = object1;
+    }
+
+    public void setObject2(V object2) {
+        this.object2 = object2;
+    }
+
+    public static <T, V> Pair of(T t, V v) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Pair.java]: try to use full name of the parameters

Comment (SviatoslavExpert) [reply src/main/java/core/basesyntax/Pair.java]: t is changed to object1, v is changed to object2

**Review by Sofasmile — COMMENTED** [review]
we can't use numbers in name of the variables or parameters
object1 -> firstObject
object2 -> secondObject
the same with setters, getters and methods

**Comment by SviatoslavExpert** [comment]
The names are changed.  Fixed. Thank you.

**Review by Sofasmile — APPROVED** [review]

**Pull request closed without merging**